astroplan/core.py

```diff
@@ -1166,6 +1166,34 @@ def can_see(self, time, target, horizon=0*u.degree, return_altaz=False):
         else:
             return observable, altaz
 
+    @u.quantity_input(horizon=u.deg)
+    def is_night(self, time, horizon=0*u.deg, obswl=None):
+        """
+        Is the Sun below ``horizon`` at ``time``?
+
+        Parameters
+        ----------
+        time : `~astropy.time.Time` or other (see below)
+            Time of observation. This will be passed in as the first argument to
+            the `~astropy.time.Time` initializer, so it can be anything that
+            `~astropy.time.Time` will accept (including a `~astropy.time.Time`
+            object)
+
+        horizon : `~astropy.units.Quantity` (optional), default = zero degrees
+            Degrees above/below actual horizon to use
+            for calculating day/night (i.e.,
+            -6 deg horizon = civil twilight, etc.)
+
+        obswl : `~astropy.units.Quantity` (optional)
+            Wavelength of the observation used in the calculation.
+        """
+        if not isinstance(time, Time):
```

Comment:
BTW: was originally going to suggest removing this line because passing a `Time` in to `Time` gets you a `Time`, but the returned `Time` is a copy of the input `Time`

`Time`
...
`Time`